tutorials/v7/draw_mt.cxx

```diff
@@ -53,25 +57,43 @@ void draw_canvas(const std::string &title, RColor col)
    }
 
    // Create a canvas to be displayed.
-   auto canvas = RCanvas::Create(title);
+   auto canvas = RCanvas::Create(title + " canvas");
    canvas->Draw(pHist)->SetLineColor(col);
+   // canvas->Draw(pHist)->SetLineColor(RColor::kRed);
    canvas->Draw(pHist2)->SetLineColor(RColor::kBlue);
 
-   canvas->Show();
 
-   for (int loop=0;loop<50;++loop) {
+   bool batch = gROOT->IsWebDisplayBatch();
+
+   int maxloop = batch ? 10 : 100;
+
+   if (title == "Zero") {
+      canvas->SaveAs("Zero.json");
+      maxloop = 1;
+   }
+
+   if (!batch) canvas->Show();
+
+   // printf("%s before loop entering\n", title.c_str());
+
+   for (int loop=0;loop<maxloop;++loop) {
```

Comment:
Add space after `;` and around `=` and  `<`.

Reply:
I just do clang-format

Reply:
Then your clang-format is broken :-( See https://github.com/root-project/root/pull/2603/files#diff-69669892fd6597c69f60f34ad54bbddeR64 where it does it just fine. Now what? Can you override clang-format here, please?

Reply:
Now is applied - I do not enable it by default, doing it by-hand for selected part of code